Lib/test/test_concurrent_futures/test_interpreter_pool.py

```diff
@@ -56,6 +56,16 @@ def pipe(self):
         return r, w
 
 
+class Shenanigans:
```

Comment:
While I like this class name, can we find a name that is a bit more explicit rather than this generic one, please? :)

Reply:
Eh, it's just for a test. It just has to be in global scope for pickle to work, I don't really want anyone else to use that. Does it matter *that* much?

Reply:
I'd say yes. I think we need to know *why* this class is special (it's not just being in the global scope; it also raises in `__init__`; if it were a random class, I wouldn't mind though). but maybe I'm over-reacting here.

Reply:
I'll name it `PickleShenanigans` to make it more clear, then.

Reply:
I'm not too worried about it.  It's easy to find where it's used and there's a comment *there* that points to the issue that describes that problem.

Reply:
Me neither, I was just humoring Bénédikt :)